.github/workflows/internal-image-publish.yaml

```
@@ -0,0 +1,20 @@
+      image_tag:
+        type: string
+        description: 'Tags for Images in GAR'
+        required: true
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)